### ISSUE TITLE
[Fixed in GH-6052] Safari: Missing visual steps and cannot tab on checkout shipping method step

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#5040]
<img width="1274" alt="Screen Shot 2019-10-16 at 1 54 44 PM" src="https://user-images.githubusercontent.com/45409515/66945190-9d47f500-f01c-11e9-9911-f5e39b552f61.png">

**Comment by Zeyber** [comment]
To QA using instructions listed here:
https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497

**Comment by KateChuen** [comment]
Tested through #4324
